src/test/migrations/steps/20240110160726-step-public-file-ddl.ts

```diff
@@ -68,7 +68,8 @@ export default async () => {
       insert into ${Schemas.getNameCycle(assessmentFRA, cycle2020)}.repository (country_iso, file_uuid, props)
       select r.country_iso,
              f.uuid,
-              props || jsonb_build_object('props', af.props, 'translation', jsonb_build_object('en', f.name))
+              props || jsonb_build_object('translation', jsonb_build_object('en', f.name))
```

Comment:
fix typo, migrated props accidentally in fra2020 to repository.props.props fixed to repository.props